go.mod

```diff
@@ -3,7 +3,7 @@ module sigs.k8s.io/aws-iam-authenticator
 go 1.22.2
 
 require (
-	github.com/aws/aws-sdk-go v1.44.332
```

Comment:
I think you'll need to update the integration tests version as well https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/09b2925e79434c40f651d55d17b33fca1220e863/tests/integration/go.mod#L6